### PR TITLE
CCTRI-1869: Fixed bug "Errors are return in 'data' field"

### PR DESCRIPTION
**Pull request by MaxMhonc**

### CODE REVIEW

File: api/utils.py [hunk 1]
```diff
@@ -70,3 +70,7 @@ def get_json(schema):
 
 def jsonify_data(data):
     return jsonify({'data': data})
+
+
+def jsonify_error(data):
+    return jsonify({'error': data})
```
Comment (GeorgiyII) [inline api/utils.py:76]: errors

File: tests/unit/api/test_authorization.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def jwt_encode(d: dict) -> str:
 def authorization_errors_expected_payload(route):
     def _make_payload_message(message):
         payload = {
-            'data': {
+            'error': {
```
Comment (GeorgiyII) [inline tests/unit/api/test_authorization.py:66]: the same

File: tests/unit/conftest.py [hunk 3]
```diff
@@ -38,7 +38,7 @@ def valid_jwt(client):
 def invalid_json_expected_payload():
     def _make_message(message):
         return {
-            'data': {
+            'error': {
```
Comment (GeorgiyII) [inline tests/unit/conftest.py:41]: the same

File: api/utils.py (hunk 1 above)
Comment (GeorgiyII) [reply api/utils.py:76]: [data] because list of errors

**Review by GeorgiyII — APPROVED** [review]

**Review by AlinaBlahun — APPROVED** [review]

**Pull request merged**